app/src/main/res/values-ru/strings.xml

```diff
@@ -19,7 +19,7 @@
     <!-- Long click menu -->
     <string name="long_click_dialog_title">Что вы хотите сделать?</string>
     <string name="long_click_open">Открыть \"%1s\"</string>
-    <string name="long_click_settings">Доступ к настройкам</string>
+    <string name="long_click_settings">Настройки</string>

```

Comment:
Although the string key just tells "settings", the English text for this string is "Access its settings".
=> Should I still apply this change? (just trying to keep things consistent across translations)